pkg/splunk/enterprise/indexercluster_test.go

```diff
@@ -145,6 +141,81 @@ func indexerClusterPodManagerTester(t *testing.T, method string, mockHandlers []
 			return c
 		},
 	}
+	return mgr, mockSplunkClient
+}
+
+// indexerClusterpodManagerVerifyRFPeersTester is used to verify replicas against RF using a indexerClusterPodManager
+func indexerClusterPodManagerVerifyRFPeersTester(t *testing.T, method string, mgr *indexerClusterPodManager,
+	desiredReplicas int32, wantPhase splcommon.Phase, wantCalls map[string][]spltest.MockFuncCall, wantError error) {
+
+	// initialize client
+	c := spltest.NewMockClient()
+
+	// test update
+	err := mgr.verifyRFPeers(c)
+	if (err == nil && wantError != nil) ||
+		(err != nil && wantError == nil) ||
+		(err != nil && wantError != nil && err.Error() != wantError.Error()) {
+		t.Errorf("%s returned error %v; want %v", method, err, wantError)
+	}
+
+	if mgr.cr.Spec.Replicas != desiredReplicas {
+		t.Errorf("spec has replicas as %d ; want %d", mgr.cr.Spec.Replicas, desiredReplicas)
+	}
+	// check calls
+	c.CheckCalls(t, method, wantCalls)
+}
+
+func indexerClusterPodManagerReplicasTester(t *testing.T, method string, mockHandlers []spltest.MockHTTPHandler,
```

Comment:
Is this API for multiple types of replica testing in the future? (In this case we are testing only RF peer replicas tester)